packages/terra-core-docs/CHANGELOG.md

```diff
@@ -13,6 +13,11 @@
   * Updated `terra-list` section guide to not use listbox role.
   * Updated `terra-divider` tests and doc examples to include heading level prop.
 
+* Added
+  * Added Accordion Examples for `terra-section-header`.
+  * Added accessibility guide for `terra-divider`
+  * Added tests and example for adding row header to `terra-html-table`.
```

Comment:
Remove duplicate entry for section-header and divider. Move html change line under existing 'Added' block

Reply:
Looks like it got duplicated while resolving merge conflicts. Fixed [5173c5f](https://github.com/cerner/terra-core/pull/3695/commits/5173c5f6fcb0a5489510757d06320deb246edd48)